components/.storybook/config.js

```diff
@@ -20,6 +20,12 @@ addDecorator(() => ({
   `
 }))
 
+addParameters({
+  options: {
+    name: 'Alpaca components'
```

Comment:
Do we want to stay with `/components` dir and title name or change it to `/ui`?

Reply:
please keep `/component` for now, we'll think it over on next alpaca meeting :)